### PR TITLE
French translation improvements

### PR DESCRIPTION
**Pull request by thedarkeye**
Just a few accents fixed and some translations improved by a bit

### CODE REVIEW

**Comment by lucas2107** [comment]
Il me semble qu'il ne faut pas d'accents

**Comment by Gunjha** [comment]
+  "pokemonNotToEvolve": "Pokémons - À ne pas évoluer",
big fail ^^
+  "pokemonNotToEvolve": "Pokémons - À évoluer",

**Comment by thedarkeye** [comment]
Fixed it, the name "notToEvolve" misled me x)

File: PokemonGo.RocketAPI.Console/Lang/france.json [hunk 1]
```diff
@@ -2,20 +2,20 @@
...
-  "maxCPtransfer": "Max. PC � transf�rer:",
-  "maxIVtransfer": "Max. IV � transf�rer:",
-  "telegramSettings": "Param�tres du Telegram",
-  "infoline1": "Ce Bot est totalement gratuit et Open-Source! Si vous avez pay�, annulez le payement.",
-  "infoline2": "Si vous avez un soucis en rapport avec Pokecrot, dites leur que le Bot a �t� vol�.",
-  "maxPokeballs": "Max. Pok� Ball:",
+  "maxCPtransfer": "Max. PC à transférer:",
+  "maxIVtransfer": "Max. IV à transférer:",
+  "telegramSettings": "Paramètres de Telegram",
+  "infoline1": "Ce Bot est totalement gratuit et Open-Source! Si vous avez payé, annulez le payement.",
```
Comment (aFlyerz) [inline PokemonGo.RocketAPI.Console/Lang/france.json]: Ce Bot est totalement gratuit et Open-Source! Si vous avez paié, annulez le paiement.

**Comment by Zodice** [comment]
Hi, there is a lot of things that aren't in french when the bot is on, where we can see pokemons. I'm french, i can help if you want.

**Comment by thedarkeye** [comment]
You can make changes to the language file or open another pull request if it's not in there

**Comment by Zodice** [comment]
And i share the translated document here ?

**Comment by thedarkeye** [comment]
@Zodice Yep

**Pull request closed without merging**